hoag/benchmark.py

```diff
@@ -194,6 +195,32 @@ def randomized_results_for_kwargs(n_random_seed=10, **kwargs):
         overall_res.append(res)
     return overall_res
 
+def framed_results_for_kwargs(n_random_seed=10, **kwargs):
+    overall_res = randomized_results_for_kwargs(n_random_seed=n_random_seed, **kwargs)
+    seed_columns = [
+        'i_iter',
+        'time',
+        'val_loss',
+        'test_loss',
+    ]
+    df_res = pd.DataFrame(columns=[
+        'seed',
+        *seed_columns,
+        *kwargs.keys(),
+    ])
+    for i_seed in range(n_random_seed):
+        df_seed = pd.DataFrame(columns=seed_columns, data=np.array([
+            range(len(overall_res.lamda_times[i_seed])),
+            overall_res.lamda_times[i_seed],
+            overall_res.val_losses[i_seed],
+            overall_res.test_losses[i_seed],
+        ]).T)
+        df_seed['seed'] = i_seed
+        for k, v in kwargs.items():
+            df_seed[k] = v
+        df_res = df_res.append(df_seed)
+    return df_res
```

Comment:
Seems overly complicated. Maybe clearer with something like this?
```python
data = [
    { 'seed': i_seed,
     'time': overall_res.lamda_times[i_seed]
     'val_loss': overall_res.val_losses[i_seed],
     'test_loss': overall_res.test_losses[i_seed],
     **kwargs
    } for i_seed in range(len(overall_res.lamda_times[i_seed]))
]
df_res = pd.DataFrame(data)
```

Reply:
done, wdyt?